app/controllers/concerns/auto_complete_for/business_unit.rb

```diff
@@ -33,6 +33,10 @@ def auto_complete_for_business_unit
     ).merge(PlanItem.allowed_by_business_units
     ).references(:business_unit_type).limit(10)
 
+    if (params.has_key? :exempt_business_unit_id) && params[:exempt_business_unit_id].present?
+      @business_units = @business_units.where('business_units.id != ?', params[:exempt_business_unit_id])
```

Comment:
```suggestion
      @business_units = @business_units.where.not(id: params[:exempt_business_unit_id])
```
Así funciona _mejor_ en los otros motores (porque el negado no suele ser igual en PotsgreSQL y Oracle por ejemplo, con esta forma lo convierte _el traductor_ del adaptador).

Reply:
impecable franco